src/credential/issuance/errors.ts

```diff
@@ -0,0 +1,47 @@
+import { IoWalletError, serializeAttrs } from "../../utils/errors";
+
+//TODO: Check for io-react-native-utils before removing them
```

Comment:
What do you think about adding the linked jira ticket to the TODO?

Reply:
I removed the TODO: in the context of this PR these errors are perfectly fine. Whether to remove them or not depends on another PR and on the merge order of the two PRs. We'll see which one gets merged first.